documentation/docs/ui/mention_list_view.md

```diff
@@ -0,0 +1,66 @@
+---
+id: uiMentionListView
+title: Mention List View
+sidebar_position: 4
+---
+
+## Creating layout
+`MentionListView` is a view that is responsible for showing previews of messages which contains current user mention.
+
+You can declare the `MentionListView` inside a layout file:
+
+```
+<io.getstream.chat.android.ui.mention.list.MentionListView
+        android:id="@+id/mentionsListView"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        />
+```
+
+| Light Mode | Dark Mode |
+| --- | --- |
+|![light](/img/mentions_list_view_light.png)|![dark](/img/mentions_list_view_dark.png)|
+
+## Adding Mention List View And Binding With View Model
+Here's an example layout containing `MentionListView`:
+```xml
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's duplicated 🙈 Maybe let's rename this section to `Binding With View Model` and remove this layout?

Reply:
I added the `Binding With View Model` section, please take a look.

Reply:
I think we misunderstood each other. Now we have duplicated `Binding With View Model` section (the second one is in  `Adding Mention List View And Binding With View Model`)
I would rather do something like that:
- Creating Layout (put there `Here's an example layout containing `MentionListView`:` together with XML)
- Binding With View Model (`Just like other views - `MentionListView` comes together with a view model which is responsible for providing necessary data:` )
- Handling Actions

Reply:
Done. But instead of **Create Layout** the section is called **Adding MentionListView**